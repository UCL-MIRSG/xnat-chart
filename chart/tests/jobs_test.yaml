templates:
  - templates/jobs.yaml
tests:
  - it: jobs manifest should match snapshot
    set:
      web.config.enabled: true
    asserts:
      - matchSnapshot:
          path: spec.template.spec
