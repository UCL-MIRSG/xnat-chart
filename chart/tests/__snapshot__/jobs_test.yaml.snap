jobs manifest should match snapshot:
  1: |
    backoffLimit: 0
    containers:
      - args:
          - configure-xnat.yaml
          - --extra-vars
          - xnat_url=http://RELEASE-NAME-xnat
          - --extra-vars
          - xnat_admin_password="$(XNAT_ADMIN_PASSWORD)"
          - --extra-vars
          - xnat_service_admin_password="$(XNAT_SERVICE_ADMIN_PASSWORD)"
          - -v
        command:
          - ansible-playbook
        env:
          - name: XNAT_ADMIN_PASSWORD
            valueFrom:
              secretKeyRef:
                key: adminPassword
                name: localdb-secret
          - name: XNAT_SERVICE_ADMIN_PASSWORD
            valueFrom:
              secretKeyRef:
                key: serviceAdminPassword
                name: localdb-secret
        image: ghcr.io/ucl-mirsg/xnat-config:latest
        name: post-install-job
    imagePullSecrets:
      - name: ""
    restartPolicy: Never
